File: 139/d.js
// const stdin = require('fs').readFileSync('/dev/stdin', 'utf8').trim()

const stdin = `1`
const input = stdin.split('\n')

const N = input[0]

const sum = num => {
  if (num === 0) return 0

  return sum(num - 1) + num
}

const ans = () => {
  const res = sum(N - 1)

  console.log(res)
}

ans()

File: 139/d.cpp
#include <bits/stdc++.h>
using namespace std;

long long sum(long long num) {
  if (num == 0) return 0;

  return sum(num - 1) + num;
}

int main() {
  cin.tie(0);
  ios::sync_with_stdio(false);

  long long N;

  cin >> N;

  long long ans = sum(N - 1);

  cout << ans << endl;

  return 0;
}